gateway/src/main/java/com/lyft/data/gateway/handler/QueryIdCachingProxyHandler.java

```diff
@@ -72,9 +73,8 @@ public String rewriteTarget(HttpServletRequest request) {
     if (!Strings.isNullOrEmpty(queryId)) {
       backendAddress = routingManager.findBackendForQueryId(queryId);
     } else {
-      String scheduledHeader =
-          request.getHeader("X-Presto-Scheduled-Query");
-      if (scheduledHeader != null && scheduledHeader.toLowerCase().equals("true")) {
+      String scheduledHeader = request.getHeader(SCHED_QUERY_HEADER);
```

Comment:
nit: This constant is just used in this line, I prefer it to be inlined.

Reply:
We can reuse this constant here https://github.com/lyft/prestoproxy/blob/master/prestoproxy-java/src/main/java/com/lyft/data/proxly/util/ProxyConnectionUtil.java#L19